react/FilterNavigator.js

```diff
@@ -103,6 +109,10 @@ const FilterNavigator = ({
     layout === LAYOUT_TYPES.mobile ||
     layout === LAYOUT_TYPES.phone
 
+  const filtersDrawerDirectionMobile =
+    DRAWER_DIRECTION_MOBILE[drawerDirectionMobile] ??
+    DRAWER_DIRECTION_MOBILE.drawerRight
```

Comment:
why the default value of the prop is `drawerLeft` and here when checking if the value is valid we use `drawerRight` as fallback?

Reply:
IMO, the code was updated due to request changes/comments and the CHANGELOG ended up forgotten.

Reply:
but what I'm referring to is the value of the prop received by the FilterNavigator which is as `drawerLeft`, not just the changelog

Reply:
Hello. I really can't remember, it was a very long time ago. Fixed now. Thank you!